examples/PACKAGES/sna_nnn_slcsa/dir.slcsa/lr_bias.dat

```diff
@@ -0,0 +1 @@
+-6.32012657 5.62127377 1.19871662 -0.49986382 
```

Comment:
Need to add a newline at end of file?

Reply:
It worked *as is*. I tweaked the input and ran it to create logs (and have the compute output to thermo data by outputting min/max of compute per-atom properties).